src/types.ts

```diff
@@ -27,37 +27,69 @@ export interface Instance {
     account: string,
     bid: Bid
   ) => Promise<boolean>;
-  isZAuctionApprovedToTransferNft: (account: string) => Promise<boolean>;
-  getZAuctionSpendAllowance: (account: string) => Promise<ethers.BigNumber>;
+  isZAuctionApprovedToTransferNft: (
+    account: string,
+    contractAddress: string
+  ) => Promise<boolean>;
+  isZAuctionApprovedToTransferNftLegacy: (
+    account: string,
+    contractAddress: string
+  ) => Promise<boolean>;
   getZAuctionSpendAllowanceByBid: (
     account: string,
     bid: Bid
   ) => Promise<ethers.BigNumber>;
-  getTradeTokenAddress: () => Promise<string>;
+  getZAuctionSpendAllowance: (
+    paymentTokenAddress: string,
+    account: string
+  ) => Promise<ethers.BigNumber>;
+  getZAuctionSpendAllowanceLegacy: (
+    account: string
+  ) => Promise<ethers.BigNumber>;
+  getDefaultPaymentToken: () => Promise<string>;
+  setDefaultPaymentToken: (
+    defaultTokenAddress: string,
+    signer: ethers.Signer
+  ) => Promise<ethers.ContractTransaction>;
+  getDomainNetworkPaymentToken: (networkId: string) => Promise<string>;
+  setDomainNetworkPaymentToken: (
```

Comment:
Rename to `Network` instead of `DomainNetwork`

Reply:
It was requested by @RyanCross that we use more specific terms to avoid overloading the term "network" which could mean the **zNS** network (0://wilder, 0://kitty, ...) but could also mean the **ethereum** network (mainnet, kovan, rinkeby)

Seems fine to reduce it to just `network` here though as we're we have context